pkg/controller/backup_sync_controller.go

```diff
@@ -232,9 +238,38 @@ func (c *backupSyncController) run() {
 			default:
 				log.Debug("Synced backup into cluster")
 			}
+
+			// process the pod volume backups from object store, if any
+			podVolumeBackups, err := backupStore.GetBackupPodVolumes(backupName)
+			if !kuberrs.IsNotFound(err) {
```

Comment:
I don't think you would ever get a kube "not found" error here, since this isn't calling the Kubernetes API. I think you just want to check for any error here, and log/continue if one is gotten